### ISSUE TITLE
LayoutLMV3 

### ISSUE THREAD

**Issue by logan-markewich** [huggingface/transformers#16949]
### Model description

LayoutLMv3 is a pre-trained multimodal Transformer for Document AI with unified text and image masking. The simple unified architecture and training objectives make LayoutLMv3 a general-purpose pre-trained model. For example, LayoutLMv3 can be fine-tuned for both text-centric tasks, including form understanding, receipt understanding, and document visual question answering, and image-centric tasks such as document image classification and document layout analysis.

LayoutLMv3 greatly simplifies training and reduces the number of parameters compared to v3, making it an important milestone in document understanding.

[LayoutLMv3: Pre-training for Document AI with Unified Text and Image Masking](https://arxiv.org/abs/2204.08387) Yupan Huang, Tengchao Lv, Lei Cui, Yutong Lu, Furu Wei, Preprint 2022.

### Open source status

- [X] The model implementation is available
- [X] The model weights are available

### Provide useful links for the implementation

[Huggingface Pretrained Download](https://huggingface.co/microsoft/layoutlmv3-base)

**Comment by NielsRogge** [comment]
Duplicate of #16914

**Comment by githublsk** [comment]
@[NielsRogge](https://github.com/NielsRogge)
I have one question for layoutlmv3, does layoutlmv3 can support RE and SER task for xfund dataset now?
